documentation/modules/adding-source-provider.adoc

```diff
@@ -176,8 +273,29 @@ If both *URL* and *Service account bearer token* are left blank, the local {ocp-
 +
 include::snip-certificate-options.adoc[]
 endif::[]
+
 ifdef::dest_vmware,dest_rhv,dest_ostack,dest_ova,dest_cnv[]
-. Click *{virt}*.
+. Access the *Create {virt} provider* interface by doing one of the following:
+
+.. In the {ocp} web console, click *Migration for Virtualization > Providers*.
+... Click *Create Provider*.
+... Select a *Project* from the list. The default project shown depends on the active project of {project-short}. 
++
+If the active project is *All projects*, then the default project is `openshift-mtv`. Otherwise, the default project is the sane as the active project.
```

Comment:
```suggestion
If the active project is *All projects*, then the default project is `openshift-mtv`. Otherwise, the default project is the same as the active project.
```